x-pack/plugins/enterprise_search/public/applications/workplace_search/views/groups/components/group_users_table.tsx

```diff
@@ -13,25 +13,14 @@ import { EuiTable, EuiTableBody, EuiTablePagination } from '@elastic/eui';
 import { Pager } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 
+import { USERNAME_LABEL, EMAIL_LABEL } from '../../../../shared/constants';
```

Comment:
Forgot to implement these in this commit. Fixed in https://github.com/elastic/kibana/pull/102826/commits/2ccb05a886149cbb3f2354f363a71ab8c289d6d3